### PR TITLE
SUS-6160 | Explicitly set timeouts for nginx - fpm communication

### PR DESCRIPTION
**Pull request by macbre**
https://wikia-inc.atlassian.net/browse/SUS-6160

http://nginx.org/en/docs/http/ngx_http_fastcgi_module.html

### CODE REVIEW

File: docker/base/nginx.conf [hunk 1]
```diff
@@ -1,7 +1,7 @@
 worker_processes  1;
 
 error_log  /var/log/nginx/error.log warn;
-pid        /var/run/nginx/nginx.pid;
+pid        /tmp/nginx.pid;
```
Comment (pwojcik86) [inline docker/base/nginx.conf]: I don't think we need to change this

**Review by pwojcik86 — CHANGES_REQUESTED** [review]

File: docker/base/nginx.conf (hunk 1 above)
Comment (macbre) [reply docker/base/nginx.conf]: We have to, for dev instance - I'm getting permission errors

Comment (pwojcik86) [reply docker/base/nginx.conf]: Then we should fix the dev configuration, not do a workaround in prod

Comment (macbre) [reply docker/base/nginx.conf]: See e09d1295c5da0f936b8202c62236a15d4884c402

**Review by hakubo — APPROVED** [review]

**Review by pwojcik86 — APPROVED** [review]

**Pull request merged**